### PR TITLE
RTL nav support on the left (bug 1148600, 1148595)

### PR DESCRIPTION
**Pull request by mstriemer**
ltr-mobile-nav-2 mov
rtl-mobile-nav-2 mov

### CODE REVIEW

File: src/media/css/elements--categories.styl [hunk 1]
```diff
@@ -49,6 +49,9 @@ mkt-category-item {
         display: inline-block;
         margin: 0 10px 0 0;
         vertical-align: middle;
```
Comment (spasovski) [inline src/media/css/elements--categories.styl]: `\n`

File: src/media/css/elements--nav.styl [hunk 2]
```diff
@@ -262,7 +262,7 @@ mkt-nav-child {
 
 .mkt-nav-root--divider {
     border: 1px solid $greyscale-white;
-    left: 10px;
+    margin: 0 10px;
```
Comment (spasovski) [inline src/media/css/elements--nav.styl]: :+1:

File: src/media/css/lib/mixins.styl [hunk 3]
```diff
@@ -186,8 +186,25 @@ category-icon-position($slug, $scale=1) {
     background-position: ($positions[0] * $cat-icon-size-default * $scale * -1) ($positions[1] * $cat-icon-size-default * $scale * -1);
 }
 
+ltr() {
```
Comment (spasovski) [inline src/media/css/lib/mixins.styl]: I tend to prefer "assume LTR until RTL overrides" but wouldn't enforce it on anyone.

Comment (mstriemer) [inline src/media/css/lib/mixins.styl]: Yeah, I try to do that also. I've been finding this helps make things a bit clearer though in some cases. I used it on the [search dropdown](https://github.com/mstriemer/fireplace/blob/3882d962d87305e05b5f672b9c04774384a6e904/src/media/css/elements--header.styl#L150-L161) and I think it looks pretty good.

File: src/media/css/elements--categories.styl (hunk 1 above)
Comment (mstriemer) [inline src/media/css/elements--categories.styl]: I'm a bit torn on this. The styles are for this element so I think it makes sense to include them right after, but I think we'd leave a blank line for something like `&.is-active` so I should probably move it down.

**Pull request merged**